src/bt2d/core/window/Window.java

```diff
@@ -119,6 +125,15 @@
      */
     public Window(int width, int height, String title, boolean fullScreen, boolean undecorated, boolean strictAspectRatio, int refreshRate)
     {
+        // set logging callbacks
+        Configuration.DEBUG_STREAM.set(createLWJGLDebugPrintStream());
+        GLFWErrorCallback.create(createGLFWErrorCallback()).set();
+
+        if (!glfwInit())
+        {
+            throw new GameContainerException("Unable to initialize GLFW");
```

Comment:
this might not be a very well suited exception as it is thrown by the window class